tests/ssl-opt.sh

```diff
@@ -1875,6 +1880,22 @@ run_test    "TLS-ECDHE-RSA Opaque key for server authentication" \
             -S "error" \
             -C "error"
 
+requires_config_enabled MBEDTLS_SSL_PROTO_TLS1_2
+requires_config_enabled MBEDTLS_USE_PSA_CRYPTO
+requires_config_enabled MBEDTLS_X509_CRT_PARSE_C
+requires_config_enabled MBEDTLS_RSA_C
+requires_config_enabled MBEDTLS_SHA256_C
+run_test    "RSA opaque key on server configured for decryption" \
```

Comment:
Minor, partially pre-existing: I think it would be good for all opaque key tests to have a description that starts with "Opaque key" - currently it's a mixture of "opaque" and "Opaque" so if we want to run all of them we have to remember and do `ssl-opt.sh -f '[Oo]paque key'` which is a bit annoying. (Also, I just find it cleaner for all tests in the same "family" to start the same and be next to each other, makes the output of the script nicer.)